cmd/tendermint/commands/flags/log_level_test.go

```diff
@@ -0,0 +1,42 @@
+package flags_test
+
+import (
+	"testing"
+
+	tmflags "github.com/tendermint/tendermint/cmd/tendermint/commands/flags"
+	"github.com/tendermint/tmlibs/log"
+)
+
+func TestIsLogLevelSimple(t *testing.T) {
+	simpleFlags := []string{"info", "debug", "some"}
+	for _, f := range simpleFlags {
+		if !tmflags.IsLogLevelSimple(f) {
+			t.Fatalf("%s is a simple flag", f)
+		}
+	}
+
+	complexFlags := []string{"mempool:error", "mempool:error,*:debug"}
+	for _, f := range complexFlags {
+		if tmflags.IsLogLevelSimple(f) {
+			t.Fatalf("%s is a complex flag", f)
+		}
+	}
+}
+
+func TestParseComplexLogLevel(t *testing.T) {
```

Comment:
Can you add a test that they work properly? Not just that they parse?

You have some nice examples of how these log levels are added... you could create a logger writing to a bytes.Buffer, then parse a log level, and send a message that should either be blocked or added, then check if it something was written to the buffer or not.